.github/workflows/main.yml

```diff
@@ -55,15 +55,35 @@ jobs:
       - name: Install Carrier
         run: |
           ./cli/dist/carrier-linux-amd64 install -n
-      # - name: Push an App
+      - name: Carrier Setup Information
+        run: |
+          ./cli/dist/carrier-linux-amd64 info
+      - name: Default Organizations
+        run: |
+          ./cli/dist/carrier-linux-amd64 orgs
+      - name: Create Organization
+        run: |
+          ./cli/dist/carrier-linux-amd64 create-org myself
+      - name: Carrier Orgs
+        run: |
+          ./cli/dist/carrier-linux-amd64 orgs
+      - name: Push an App
+        run: |
+          sleep 8m ;# Wait for tekton to be usable
```

Comment:
Shouldn't we better add waiting loops in the install command to make sure Tekton is usable? We can do it on another PR if you prefer.

Reply:
I agree that having some kind of waiting in the installer would be best. I have however no idea wait exactly we have to wait for in tekton, so for this one I simply did the hack to get around whatever this is. A new PR changing tekton (and removing the sleep here) would indeed be my preference.

Reply:
ok let's do it like that